to-time/index.d.ts

```diff
@@ -0,0 +1,163 @@
+         * @param {number} number of seconds
+         * @return {TimeFrame} the instance
+        */
+        addSeconds(seconds: number): TimeFrame;
+
+        /**
+         * Append minutes to the TimeFrame value
+         * @param {number} number of minutes
+         * @return {TimeFrame} the instance
+        */
+        addMinutes(minutes: number): TimeFrame;
+
+        /**
+         * Append hours to the TimeFrame value
+         * @param {number} number of hours
+         * @return {TimeFrame} the instance
+        */
+        addHours(hours: number): TimeFrame;
+
+        /**
+         * Append days to the TimeFrame value
+         * @param {number} number of days
+         * @return {TimeFrame} the instance
+        */
+        addDays(days: number): TimeFrame;
+
+        /**
+         * Append weeks to the TimeFrame value
+         * @param {number} number of weeks
+         * @return {TimeFrame} the instance
+        */
+        addWeeks(weeks: number): TimeFrame;
+
+        /**
+         * Append years to the TimeFrame value
+         * @param {number} number of years
+         * @return {TimeFrame} the instance
+        */
+        addYears(years: number): TimeFrame;
+
+        /**
+         * Transforms the TimeFrame object into human readable string representation
+         * e.g: `toTime('36930 hours').humanize()` would become "4 Years, 11 Weeks, 1 Day, 18 Hours"
+         * @return {string} human readable string representation
+        */
+        humanize(): string;
+    }
+
+    export interface toTime {
+
+
+
+
+        (data: string): TimeFrame;
+        
+        /**
+         * Initialize a new TimeFrame object with milliseconds 
+         * @return {TimeFrame} new TimeFrame instance
+        */
+        fromMilliseconds(milliseconds: number): TimeFrame;
+
+        /**
+         * Initialize a new TimeFrame object with seconds 
+         * @return {TimeFrame} new TimeFrame instance
+        */
+        fromSeconds(seconds: number): TimeFrame;
+
+        /**
+         * Initialize a new TimeFrame object with minutes 
+         * @return {TimeFrame} new TimeFrame instance
+        */
+        fromMinutes(minutes: number) : TimeFrame;
+
+        /**
+         * Initialize a new TimeFrame object with hours 
+         * @return {TimeFrame} new TimeFrame instance
+        */
+        fromHours(hours: number) : TimeFrame;
+
+        /**
+         * Initialize a new TimeFrame object with days 
+         * @return {TimeFrame} new TimeFrame instance
+        */
+        fromDays(days: number) : TimeFrame;
+
+        /**
+         * Initialize a new TimeFrame object with weeks 
+         * @return {TimeFrame} new TimeFrame instance
+        */
+        fromWeeks(weeks: number) : TimeFrame;
+
+        /**
+         * Initialize a new TimeFrame object with years 
+         * @return {TimeFrame} new TimeFrame instance
+        */
+        fromYears(years: number) : TimeFrame;
+    }
+
+}
+
+declare module 'to-time' {
```

Comment:
could you stripe `declare module 'to-time' {`?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred